java/rocksjni/env_flink.cc

```diff
@@ -20,8 +20,7 @@
 
 #include <jni.h>
 
-#include <vector>
-
+#include "include/org_rocksdb_FlinkEnv.h"
```

Comment:
It looks like this file doesn't exist? correct me If I'm wrong.

Reply:
The header file should be generated manually or by make command as you could see other source files in rocksjni.